src/main/java/com/mraof/minestuck/data/loot_table/MSBlockLootTables.java

```diff
@@ -832,6 +832,25 @@ protected void generate()
 		dropSelf(END_SAPLING.get());
 		dropSelf(SHADEWOOD_SAPLING.get());
 		
+		dropPottedContents(POTTED_FROST_SAPLING.get());
+		dropPottedContents(POTTED_RAINBOW_SAPLING.get());
+		dropPottedContents(POTTED_END_SAPLING.get());
+		dropPottedContents(POTTED_SHADEWOOD_SAPLING.get());
+		
+		dropPottedContents(AspectTreeBlocks.POTTED_BLOOD_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_BREATH_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_DOOM_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_HEART_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_HOPE_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_LIFE_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_LIGHT_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_MIND_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_RAGE_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_SPACE_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_TIME_ASPECT_SAPLING.get());
+		dropPottedContents(AspectTreeBlocks.POTTED_VOID_ASPECT_SAPLING.get());
```

Comment:
Corrected and added `dropPottedPlant()` since AspectTreeBlockData can't use `dropPottedContents()` directly. I updated the loot entries for the 'regular' saplings as well.